docker/README.md

```diff
@@ -0,0 +1,6 @@
+# Docker
+
+Currently only a single, official docker container
+(`matrixdotorg/sydent-build`) is provided. It is intended to set up a running
+environment for Sydent and nothing more. You can find it [on Docker
```

Comment:
```suggestion
environment for testing Sydent and nothing more. You can find it [on Docker
```

Just so that we don't accidentally end up supporting a sydent docker image